modules/prokka/prokka.nf

```diff
@@ -16,6 +17,32 @@ process PROKKA {
         prokka_reference_argument = "--proteins ${params.prokka_reference}"
     }
 
+    stain = sketch_string.split(",")[0]
+    genus = sketch_string.split(",")[1]
+    species = sketch_string.split(",")[2]
+
+    prokka_gramstain_argument = ""
+    prokka_genus_argument = ""
+    prokka_species_argument = ""
+
+    if (stain == "pos"):
+	prokka_gramstain_argument = "--gram pos"
+    elif (stain == "neg"):
+        prokka_gramstain_argument = "--gram neg"
+    else:
+        prokka_gramstain_argument = ""
+
+    if (genus != "Multiple"):
+        prokka_genus_argument = "--genus "+genus
+        prokka_species_argument = "--species "+species
+    else:
+        prokka_genus_argument = "--genus Multiple_taxa"
+
+    if (stain == "Not_in_list"):
+        print("Genus not found in referencelist and remains unstained!")
+    elif (stain == "Contaminated"):
+        print("Sample contains more than one genus!")
```

Comment:
@emilio-r I've removed this part since

1. It won't show up in nextflow logs 

2. The output isn't used anywhere in downstream processes.